homeassistant/components/device_automation/__init__.py

```diff
@@ -0,0 +1,68 @@
+"""Helpers for device automations."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components import websocket_api
+from homeassistant.core import split_entity_id
+from homeassistant.helpers.entity_registry import async_entries_for_device
+from homeassistant.loader import async_get_integration, IntegrationNotFound
+
+DOMAIN = 'device_automation'
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass, config):
+    """Set up device automation."""
+    hass.components.websocket_api.async_register_command(
+        websocket_device_automation_list_triggers)
+    return True
+
+
+async def async_get_device_automation_triggers(hass, device_id):
+    """List device triggers."""
+    device_registry = await hass.helpers.device_registry.async_get_registry()
```

Comment:
Run both at the same time: `dev_reg, ent_reg = await asyncio.gather(…)`

Reply:
Fixed.